Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -414,9 +460,13 @@ namespace Terrain
 
     AZStd::vector<Physics::HeightMaterialPoint> TerrainPhysicsColliderComponent::GetHeightsAndMaterials() const
     {
-        AZStd::vector<Physics::HeightMaterialPoint> heightMaterials;
-        GenerateHeightsAndMaterialsInBounds(heightMaterials);
+        int32_t gridWidth = 0, gridHeight = 0;
```

Comment:
uber nit: I think the o3de coding standards recommends to avoid doing more that one thing in an statement to simplify debugging. Recommended to use 2 lines.